pkg/kotsadmsnapshot/backup.go

```diff
@@ -100,10 +100,6 @@ func CreateApplicationBackup(ctx context.Context, a *apptypes.App, isScheduled b
 	includedNamespaces := []string{appNamespace}
 	includedNamespaces = append(includedNamespaces, kotsKinds.KotsApplication.Spec.AdditionalNamespaces...)
 
-	if os.Getenv("KOTSADM_ENV") == "dev" {
```

Comment:
this intentional?

Reply:
yeah, kotsadm is not included in application backups